File: beecrowd -  1040 - Média 3.cpp
#include <iostream>
using namespace std;
int main() {
  float a,b,c,d,media,exame;
  cin >> a >> b >> c >> d;
  media = (a*2+b*3+c*4+d)/10;
  cout.precision(1);
  if(media>=7){
    cout << fixed << "Media: " << media << endl << "Aluno aprovado." << endl;
  }
  else if(media < 5){
    cout << fixed << "Media: " << media << endl << "Aluno reprovado." << endl;
  }
  else{
    cout << fixed << "Media: " << media << endl << "Aluno em exame." << endl;
    cin >> exame;
    media = (media+exame)/2;
    cout << fixed << "Nota do exame: " << exame << endl;
    if (media >= 5){
      cout << fixed << "Aluno aprovado." << endl << "Media final: " << media << endl;
    }
    else{
      cout << fixed << "Aluno reprovado." << endl << "Media final: " << media << endl;
    }
  
 
  }

  return 0;
}

File: beecrowd -  1040 - Média 3.py
x,y,d,e = map(float,input().split())
ponderada = ((2*x) +   (3* y) + (4*d) +  (1*e)) / 10
if ponderada >= 7:
  print('Media: %.1f\nAluno aprovado.'%ponderada)
elif ponderada < 5:
    print('Media: %.1f\nAluno reprovado.'%ponderada)
else:
  print('Media: %.1f\nAluno em exame.'%ponderada)
  nova = float(input())
  if (nova+ponderada/2) >= 5:
    print('Nota do exame: %.1f\nAluno aprovado.\nMedia final: %.1f'%(nova,((nova+ponderada)/2)))
  else:
    print('Nota do exame: %.1f\nAluno reprovado.\nMedia final: %.1f'%(nova,((nova+ponderada)/2)))
